index.js

```diff
@@ -1,29 +1,51 @@
 module.exports = {
-  extends: ['eslint-config-airbnb'].map(require.resolve),
+  env: {
+    'jest/globals': true,
+    browser: true,
+  },
+  extends: [
+    'eslint-config-airbnb',
+    'eslint-config-prettier',
+    'eslint-config-prettier/react',
+  ].map(require.resolve),
   parser: 'babel-eslint',
-  plugins: [
-    "mocha"
-  ],
+  plugins: ['jest', 'prettier'],
   rules: {
     'arrow-body-style': 0,
     'arrow-parens': ['error', 'as-needed'],
-    'camelcase': ['error'],
+    camelcase: ['error'],
+    'no-prototype-builtins': 0,
+    'import/prefer-default-export': 0,
+    'import/no-named-default': 0,
+    'import/first': 0,
     'import/no-unresolved': 0,
-    'max-len': 0,
-    'no-param-reassign': 0,
-    'no-underscore-dangle': [2, { 'allowAfterThis': true }],
-    'no-unused-vars': ['error', { vars: 'all', args: 'after-used' }],
+    'import/no-extraneous-dependencies': 0,
+    'no-underscore-dangle': [2, { allowAfterThis: true }],
+    semi: [2, 'never'],
+    'no-plusplus': ['warn'],
     'react/jsx-no-bind': 0,
     'react/prefer-stateless-function': 0,
     'react/sort-comp': 0,
-    'semi': [2, 'never'],
-    "mocha/no-exclusive-tests": "error",
+    'react/jsx-filename-extension': 0,
+    'jest/no-disabled-tests': 'warn',
```

Comment:
why warn?  we should error for this.

Reply:
for disabled? What if we want to disable something temporarily? you think we should just always delete and add back?

Reply:
> for disabled? What if we want to disable something temporarily? you think we should just always delete and add back?

Depends...

First, IMO disabling a test should very rarely be allowed.  

One situation warranting this is a hotfix.  (Assuming hotfix deployment ends up going through the same linting) I would rather have someone add a eslint ignore comment block.  It adds some extra friction, making sure its deliberate.  For consumer, I'll try to enforce Jira tickets in the comment as well. 

Otherwise, yes, I think they should be removed and added back.  Skipped tests quickly become invalid and are not maintained.  I'd rather have someone justify removal, then say "I'm just temporarily disabling it", because the truth is we will be missing that test coverage that was there before.

Reply:
sure. im fine with this, will change